BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Public/BugsnagPerformance.cs

```diff
@@ -159,17 +160,13 @@ private void Start()
             _appStartHandler.Start();
             SetupNetworkListener();
             SetupSceneLoadListeners();
-            
             IsStarted = true;
         }
 
         private void OnSpanEnd(Span span)
         {
             _potentiallyOpenSpans.Remove(span);
-            if (!span.WasAborted)
-            {
-                _tracer.OnSpanEnd(span);
-            }
+            _tracer.OnSpanEnd(span);
```

Comment:
Does this mean we run callbacks if the span is already aborted? Or can that not happen?